compilers/concrete-compiler/compiler/tests/end_to_end_tests/end_to_end_jit_aes_short.cc

```diff
@@ -2692,7 +2694,7 @@ module  {
 )XXX",
              "main", false, true, false);
 
-  std::vector<uint8_t> input0 = {
+  std::vector<uint64_t> input0 = {
```

Comment:
Why did this change from 8 to 64 bits?

Reply:
Before the refactoring, there was a template based mechanism that allowed any precision to be used for ciphertext inputs. This is not replicated in this version. Ciphertext inputs must be `uint64_t` or `int64_t` depending on the signedness.

See https://github.com/zama-ai/concrete/blob/54e0c2251fa080dcb969bc095423f170d0cef89d/compilers/concrete-compiler/compiler/lib/Common/Transformers.cpp#L261-L266